modules/otel-config-send-metrics-monitoring-stack.adoc

```diff
@@ -6,12 +6,7 @@
 [id="configuration-for-sending-metrics-to-the-monitoring-stack_{context}"]
 = Configuration for sending metrics to the monitoring stack
 
-One of two following custom resources (CR) configures the sending of metrics to the monitoring stack:
-
-* OpenTelemetry Collector CR
-* Prometheus `PodMonitor` CR
-
-A configured OpenTelemetry Collector CR can create a Prometheus `ServiceMonitor` CR for scraping the Collector's pipeline metrics and the enabled Prometheus exporters.
+As shown in the next example, you can configure the `OpenTelemetryCollector` custom resource (CR) to create a Prometheus `ServiceMonitor` CR, or a `PodMonitor` CR for a sidecar deployment, for scraping the Collector's pipeline metrics and the enabled Prometheus exporters.
```

Comment:
I do not think the "As shown in the next example" adds anything here. The example title states this. I'd remove this transitional phrase so the focus is on what action the user can take.